index.html

```diff
@@ -0,0 +1,153 @@
+                    <li class="nav-list"><a href="about.html">About</a></li>
+                    <li class="nav-list"><a href="#mainprogramme">Program</a></li>
+                    <li class="nav-list"><a href="#">Join</a></li>
+                    <li class="nav-list"><a href="#">Sponsor </a></li>
+                    <li class="nav-list"><a href="#">News</a></li>
+                </ul>
+            </nav>
+
+        </section>
+
+        <article class="sec2">
+            <div class="mp" id="mainprogramme">
+                <h2 class="mainprogramme">Main Programme</h2>
+            </div>
+            <div class="programmecontainer">
+                <div class="cont">
+                    <img src="images/lecturer.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">lectures</h3>
+                    <div class="cont-description">Come get a talk from the most
+                        professional perspectives in the globe</div>
+                </div>
+                <div class="cont">
+                    <img src="images/exhibition.jpg" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Exhibition</h3>
+                    <div class="cont-description">Appreciate various art
+                        excellence on display organized from across
+                        continents</div>
+                </div>
+                <div class="cont">
+                    <img src="images/forum.jpg" alt="lecturer icon" width="70px"
+                        height="auto">
+                    <h3 class="cont-text">Forum</h3>
+                    <div class="cont-description">Come have the time to share
+                        your thoughts or questions with experts on
+                        whose answers you can rely </div>
+                </div>
+                <div class="cont">
+                    <img src="images/workshop.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Workshop</h3>
+                    <div class="cont-description">There will be a platformwhere
+                        you can build your own work whilst drawing
+                        inspiration from the live scenery</div>
+                </div>
+                <div class="cont">
+                    <img src="images/extra-badge-xxl.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Extras</h3>
+                    <div class="cont-description">There will be prizes to go
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
+
+        </article>
+
+
+        <section class="sec3">
+            <div class="guestscontainer">
+
+            </div>
+        </section>
```

Comment:
Kindly fix the font-style in this section to match the given design and add some spacing around the section too. Also kindly note that the number of featured elements should be 6, currently it's 4 in your design, so kindly add 2 more 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190651455-0af47e42-cf1a-4d84-b5cd-2b8e032b0f51.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190651593-af8ec545-f803-48d1-b987-75dca523a701.png)

Reply:
done, thanks